### PR TITLE
feat(Header): Implement an appbar on all page

### PR DESCRIPTION
**Pull request by FabienHP**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
-import { Navigate, Route, Routes } from 'react-router-dom';
+import { CssBaseline } from '@mui/material';
 import { useSelector } from 'react-redux';
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
+import Header from './components/Header/Header';
 
 function App() {
   const { message } = useSelector(state => state.init)
```
Comment (remirubis) [inline src/App.js:8]: Are you configured Eslint? It seems that there are not the necessary rules.

**Review by remirubis — CHANGES_REQUESTED** [review]
Few changes :)

File: src/App.js (hunk 1 above)
Comment (remirubis) [reply src/App.js:8]: Line 8 -> Missing comma

**Review by remirubis — DISMISSED** [review]
LGTM

**Review by remirubis — DISMISSED** [review]

**Comment by remirubis** [comment]
For working on another ticket I made a rebase 😄

**Review by remirubis — APPROVED** [review]

**Review by kilianpichard — APPROVED** [review]
LGTM !

**Pull request merged**